cocos/storage/local-storage/LocalStorage.h

```diff
@@ -46,7 +46,7 @@ void CC_DLL localStorageFree();
 void CC_DLL localStorageSetItem( const std::string& key, const std::string& value);
 
 /** Gets an item from the JS. */
-std::string CC_DLL localStorageGetItem( const std::string& key );
+bool CC_DLL localStorageGetItem( const std::string& key, std::string *outItem );
```

Comment:
The old API and implementation is wrong. It can't differentiate the situation when item doesn't exist and when item's value is empty string.
And the result is also confirmed different from the web localStorage result.